### DISCUSSION TITLE
tle missing from pip install

### DISCUSSION THREAD

**Discussion by joeskeens** [bradsease/oem#77] [General]
Hi there, I'm trying to use this package to convert TLE files to OEM format. I installed with pip, but the tle name seems to be missing. 
```
from oem import tle
*** ImportError: cannot import name 'tle' from 'oem'
```
Is there a reason why this script isn't included in the tagged versions?

Thanks,
Joe

**Comment by bradsease** [comment]
Hi Joe,

Sorry about that! I thought I had rolled that into a version but I guess I missed it. I'm working on a new version soon with a revised parser and I'll make sure it gets in there. In the meantime,  you can install the latest version of the `main` branch here and the TLE feature should work
